docs/docs-beta/docs/guides/data-modeling/asset-factories.md

```diff
@@ -1,5 +1,73 @@
 ---
-title: "Creating asset factories"
+title: 'Creating domain-specific languages with asset factories'
 sidebar_position: 60
-sidebar_label: "Creating asset factories"
+sidebar_label: 'Creating domain-specific languages'
```

Comment:
I think 'Asset Factories' is something people would look up more often than 'domain-specifc languages'. Any reason why you went with the new sidebar title?

Reply:
I thought it was the opposite. Isn't "Asset factories" specific to Dagster? To me it doesn't seem like a customer job to be done. I'm not super convinced though so plz let me know if I should change it and I can revert.

Reply:
I don't know if there's an easy answer. Are Asset Factories always in service of DSLs? If so -- let's leave it. Search will also help us here.